ignite/metrics/precision.py

```diff
@@ -15,11 +15,11 @@ def __init__(self, output_transform=lambda x: x, average=False, is_multilabel=Fa
         self.eps = 1e-20
 
     def reset(self):
-        self._true_positives = torch.DoubleTensor(0) if self._is_multilabel else 0
-        self._positives = torch.DoubleTensor(0) if self._is_multilabel else 0
+        self._true_positives = torch.DoubleTensor(0) if (self._is_multilabel and not self._average) else 0
+        self._positives = torch.DoubleTensor(0) if (self._is_multilabel and not self._average) else 0
 
     def compute(self):
-        if not isinstance(self._positives, torch.Tensor):
+        if not (isinstance(self._positives, torch.Tensor) or self._positives > 0):
```

Comment:
@anmolsjoshi could you please add a test to raise `NotComputableError` in two cases of multilabel. I think we missed `or self._positives > 0` in the current implementation

Reply:
Done!